cmd/controller/controller.go

```diff
@@ -68,7 +68,7 @@ func main() {
 
 	// watch pvcs in all namespaces
 	pvcListWatcher := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "persistentvolumeclaims", "", fields.Everything())
-	cdiController := controller.NewController(client, queue, pvcInformer, pvcListWatcher, importerTag)
+	cdiController := controller.NewController(client, queue, pvcInformer, pvcListWatcher, importerImage)
```

Comment:
_pkg/controller/controller.go_'s NewController func should name this arg "importerImage" rather than "importerImageTag"